### DISCUSSION TITLE
3d models rotations

### DISCUSSION THREAD

**Discussion by redobsi** [rocksdanister/lively#2360] [Ideas]
https://github.com/user-attachments/assets/40f21db9-5247-40ce-ae59-f2557648b42e
![image](https://github.com/user-attachments/assets/dd03b420-0db5-4d66-adf9-8ca01200aa7a)

Check the video and the image please.

This is just a basic 3d model loader with matrix multiplication for the rotation and some neat texturing, here in the example it's just a cube rgb randomizer following sin value, you can change the speed and the FOV (field of view) atm. 

This is a wallpaper I created for fun and I think it may be a good add on to the app, I can send the source straight away

**Comment by rocksdanister** [comment]
Its a good start but too simple at the moment.

You can try playing around with the idea for example the entire screen is a cube and it has a blurred albumart:
https://github.com/rocksdanister/lively/wiki/Web-Guide-V-:-System-Data
